reactive/kotlinx-coroutines-rx2/test/LeakedExceptionTest.kt

```diff
@@ -22,37 +22,84 @@ class LeakedExceptionTest : TestBase() {
 
     @Test
     fun testSingle() = withExceptionHandler(handler) {
-        val flow = rxSingle<Unit> { throw TestException() }.toFlowable().asFlow()
-        runBlocking {
-            repeat(10000) {
-                combine(flow, flow) { _, _ -> Unit }
-                    .catch {}
-                    .collect { }
+        withFixedThreadPool(4) { dispatcher ->
+            val flow = rxSingle<Unit>(dispatcher) { throw TestException() }.toFlowable().asFlow()
+            runBlocking {
+                repeat(10000) {
+                    combine(flow, flow) { _, _ -> Unit }
+                            .catch {}
+                            .collect { }
```

Comment:
Style nit: Continuation lines should be formatted with 4 spaces of indentation (not 8).
Also, it as we are touching this code anyway, it would be create to fix `{ }` -> `{}` (remove space)

Reply:
Fixed.